tests/asyncio_fixture.py

```diff
@@ -43,6 +43,7 @@ def fake_user_events(session):
     create_event(
         db=session,
         title='Cool today event',
+        color='red',
```

Comment:
Not sure I got that, are the colors related to your feature?

Reply:
No. While coding, 30 tests were failed because "color" column added to User schema and was pushed but wasn't supported by all tests written so far. Fixed it and added colors to some tests to make sure everything works.